Sources/Models/Postman/Postman.swift

```diff
@@ -0,0 +1,83 @@
+// Generated by https://quicktype.io
+//
+// To change quicktype's target language, run command:
+//
+//   "Set quicktype target language"
+
+import Foundation
+
+struct PostmanCollection: Codable {
```

Comment:
I think it might be useful to add a comment with an example of what the final JSON should look like.

Reply:
The final JSON is really big. Do you like to add an example as a JSON file?

Reply:
Yes, it sounds good. 🙏